highfidelity/hifi
# Possible fix for QML texture leak

I *think* this solves: https://highfidelity.manuscript.com/f/cases/13422/Teleporting-sometimes-locks-up-a-user-s-PC

Test plan:
- First, using master, go to an empty domain where you have edit rights.  Open the stats and take note of your "GPU Free Memory" and "QML Texture Memory" in the fourth column.
- Run [this](https://gist.githubusercontent.com/SamGondelman/f156d615fbe75831343994cede0f1c3b/raw/0f8db6f29d543bca432715d31c88a4905d761c77/WebEntityTest2.js).  It repeatedly creates and destroys a bunch of web entities in front of you.
- Look at the stats *WHEN THERE ARE NO WEB ENTITIES* (you should have ~5 seconds).  Over time, you should see "GPU Free Memory" go down and "QML Texture Memory" go up.
- Repeat with this PR.  Over time, you should see "GPU Free Memory" and "QML Texture Memory" stay the same.
